browser/ui/webui/settings/brave_privacy_handler.cc

```diff
@@ -8,11 +9,18 @@
 
 #include "base/bind.h"
 #include "base/values.h"
+#include "brave/common/pref_names.h"
 #include "chrome/common/pref_names.h"
 #include "chrome/browser/profiles/profile.h"
+#include "components/gcm_driver/gcm_buildflags.h"
 #include "components/prefs/pref_service.h"
 #include "content/public/common/webrtc_ip_handling_policy.h"
 #include "content/public/browser/web_ui.h"
+#include "content/public/browser/web_ui_data_source.h"
+
+#if !BUILDFLAG(USE_GCM_FROM_PLATFORM)
+#include "components/gcm_driver/gcm_channel_status_syncer.h"
```

Comment:
we probably should add deps to GN for this and buildflag

Reply:
done: https://github.com/brave/brave-core/pull/3390/files#diff-4298e412eb456ad3ceed2346abb95f6a